tests/e2e/e2e_test.go

```diff
@@ -421,38 +431,38 @@ var _ = ginkgo.Describe("[Start/Remove/Restart/Add/Stop]", func() {
 	time.Sleep(10 * time.Second)
 	ginkgo.It("can add a node", func() {
 		ginkgo.By("calling AddNode", func() {
-			color.Outf("{{green}}calling 'add-node' with the valid binary path:{{/}} %q\n", execPath1)
+			log.Info(logging.Green.Wrap("calling 'add-node' with the valid binary path: %q"), execPath1)
 			ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
 			resp, err := cli.AddNode(ctx, newNodeName, execPath1)
 			cancel()
 			gomega.Ω(err).Should(gomega.BeNil())
-			color.Outf("{{green}}successfully started:{{/}} %+v\n", resp.ClusterInfo.NodeNames)
+			log.Info(logging.Green.Wrap("successfully started: %+v"), resp.ClusterInfo.NodeNames)
```

Comment:
probably you should use %q everywhere for string slices (by the way, nice to know %q can be used for that)